htdocs/admin/pdf_other.php

```diff
@@ -103,6 +103,25 @@
 }
 print '</td></tr>';
 
+print '<tr class="oddeven"><td>'.$langs->trans("PDF_NO_SENDER_FRAME");
```

Comment:
Because this is a global option, you must removed it from this page (it has already been added on previous page)

Reply:
@eldy it's done